### PR TITLE
Replace placeholder block with participation rate

### PR DESCRIPTION
**Pull request by Ploypiti**
Thank you for your pull request! Please read and understand everything below

Please acknowledge that you have read and understood each of the following before submitting your pull request.

# _Do not_ delete any text other than where you are instructed.

**Maintainers: If any of these are left unchecked, do not merge.**

**Students: If one of them is not applicable to you -- Please check it anyways. DO NOT REMOVE ANYTHING!**

**You may check them after opening your pull request.**

You may do so by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [x] Read and understood (see CONTRIBUTING.md)
- [x] Included a live link or preview screenshot
- [x] Images are `240 x 240` [w x h]
- [x] Resolved merge conflicts
- [x] Included a description of my change below

# Things done in this Pull Request

- Preview https://ploypiti.github.io/gci.fossasia.org/
- Replace placeholder block (The one with 2008, 150+, Millions, etc.) with students, tasks, and mentors amount
- #269 point 12

### CODE REVIEW

**Comment by anshumanv** [comment]
Please provide a test link where we can see your changes. :smile:

**Review by anshumanv — APPROVED** [review]

**Review by isuruAb — COMMENTED** [review]
Hi @Ploypiti , Please squash you 3 commits

**Comment by Ploypiti** [comment]
@isuruAb Done, but I'm not sure I got it the right way.

File: index.html [hunk 1]
```diff
@@ -297,21 +297,21 @@ <h6 class="uppercase mb16">A timeless design to be proud of</h6>
 	<div class="container">
 		<div class="row">
 			<div class="col-sm-4">
-				<h2 class="number">2008</h2>
+				<h2 class="number">370+ students</h2>
 				<p>
-					At vero eos et accusamus et iusto odio dignissimos ducimus qui blanditiis praesentium voluptatum deleniti atque corrupti quos dolores et quas molestias excepturi sint occaecati cupiditate non provident.
+					Over 30 students participated in Fossasia Google code-in 2017, many students made their way in contributing on many open source projects.
```
Comment (ParthS007) [inline index.html]: @Ploypiti Please correct no of students . I don't think it's only 30.

**Comment by Ploypiti** [comment]
@ParthS007 Done 😊

**Review by ParthS007 — APPROVED** [review]
LGTM ! Good work 💯

**Comment by ParthS007** [comment]
@Ploypiti from next time ,try to squash commits ,it will be a plus for you .

**Pull request merged**